src/assisted_test_infra/test_infra/utils/utils.py

```diff
@@ -449,20 +449,24 @@ def get_kubeconfig_path(cluster_name: str) -> str:
 
 
 def get_default_openshift_version(client=None) -> str:
-    if client:
-        log.info("Using client to get default openshift version")
-        ocp_versions_dict = client.get_openshift_versions()
-        versions = [k for k, v in ocp_versions_dict.items() if v.get("default", False)]
-    else:
-        log.info(f"Reading {consts.RELEASE_IMAGES_PATH} to get default openshift version")
-        with open(consts.RELEASE_IMAGES_PATH, "r") as f:
-            release_images = json.load(f)
-            versions = [v.get("openshift_version") for v in release_images if v.get("default", False)]
-
-    log.info(f"Default openshift version found {versions}")
-
-    assert len(versions) <= 1, f"There should be no more than one default version - {versions}"
-    return versions[0] if versions else None
+    try:
+        if client:
+            log.info("Using client to get default openshift version")
+            ocp_versions_dict = client.get_openshift_versions()
+            versions = [k for k, v in ocp_versions_dict.items() if v.get("default", False)]
+        else:
+            log.info(f"Reading {consts.RELEASE_IMAGES_PATH} to get default openshift version")
+            with open(consts.RELEASE_IMAGES_PATH, "r") as f:
+                release_images = json.load(f)
+                versions = [v.get("openshift_version") for v in release_images if v.get("default", False)]
+
+        log.info(f"Default openshift version found {versions}")
+
+        assert len(versions) <= 1, f"There should be no more than one default version - {versions}"
+        return versions[0] if versions else None
+    except Exception as e:
```

Comment:
this `except` is supposed to catch the `assert` above?

if it's the case, I think introducing a `if` would be better as it is an expected behavior.

Reply:
I do think that we might want not to catch all exceptions only because that it could cause us miss corrupted release images configurations